packages/contentstack-seed/package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "@contentstack/cli-cm-seed",
   "description": "create a Stack from existing content types, entries, assets, etc.",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "author": "Contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
-    "@oclif/core": "^1.21.0",
     "@contentstack/cli-cm-import": "^1.3.0",
     "@contentstack/cli-command": "^1.1.0",
```

Comment:
@aman19K cli-command is at it's latest verion 1.2.0, why you are still using 1.1.0?

Reply:
@netrajpatel version updated.